src/cli.rs

```diff
@@ -114,6 +123,10 @@ fn daemon(matches: &ArgMatches) {
     let api_port = matches.value_of("API_PORT").unwrap().parse().unwrap();
     let server_port = matches.value_of("P2P_PORT").unwrap().parse().unwrap();
     let tor_active: bool = matches.value_of("TOR_ACTIVE").unwrap().parse().unwrap();
+    let level: String = matches.value_of("LOG_LEVEL").unwrap().parse().unwrap();
+    let env = Env::default()
+        .filter_or("RUST_LOG", level);
```

Comment:
This line break looks strange.
Can you run it through [rustfmt](https://github.com/rust-lang/rustfmt)? I suggest you add auto-formatting with rustfmt to your editor. I use [rust.vim](https://github.com/rust-lang/rust.vim) plugin for my editor. I'm sure its possible for other editors as well.

Reply:
Oops, sorry – I was under the impression it formats on save, turned out this setting was off. Fixed.